examples/c/ex_verbose.c

```diff
@@ -64,8 +64,8 @@ config_verbose(void)
     event_handler.handle_general = NULL;
 
     /*! [Configure verbose_messaging] */
-    error_check(wiredtiger_open(
-      home, (WT_EVENT_HANDLER *)&event_handler, "create,verbose=[api:1,version,write:0]", &conn));
+    error_check(wiredtiger_open(home, (WT_EVENT_HANDLER *)&event_handler,
+      "create,verbose=[api:1,all:0,version,write:2]", &conn));
```

Comment:
@ajmorton, it seems weird that we need to pass `all` now. I believe it should be `0` by default if unspecified.
@y123456yz, was this change mandatory for the test to pass, is that why you changed it?

Reply:
no, I add it only to remind beginners of this feature,  because beginners  usually use the examples to study,  if enable all verbos, this can help them to sudy.

the level 0(all:0) is only an example.

@etienneptl  thanks